README.md

```diff
@@ -41,7 +41,13 @@ Right now, the developer experience is not optimal due to 0.11 not really suppor
 
 If you want to test your changes, you gotta to the following:
 
-**Step 1:** Create a deployment for local usage:
+**Step 1:** Install required python pacakges:
+
+```sh-session
+[user@host] microzig-monorepo/ $ pip3 install -r tools/requirements.txt
+```
```

Comment:
Can you show that with creating a venv instead of just installing it systemwide? Or show both options?

Reply:
I'll do it myself, thanks for the contribution!